package.json

```diff
@@ -84,7 +84,10 @@
 		},
 		"devPlugins": [
 			"@oclif/plugin-help"
-		]
+		],
```

Comment:
Added a new hook to initialize the ioc container. See the file for a comment on hooks.